service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/FurtherEvidenceUploadedEventHandlerTest.java

```diff
@@ -235,7 +247,7 @@ void shouldSendNotificationWhenConfidentialRespondentStatementsIsUploadedByLA()
             userDetailsLA(), DESIGNATED_LOCAL_AUTHORITY, EMPTY_CASE_DATA_MODIFIER,
             (caseData) ->  caseData.getRespondentStatements().addAll(
                 buildRespondentStatementsList(buildConfidentialDocumentList(LA_USER))),
-            Set.of(DocumentUploadNotificationUserType.ALL_LAS),
+            Set.of(ALL_LAS, CAFCASS),
```

Comment:
Is this saying that cafcass is going to get emails for confidential respondent statements?